$NetBSD: patch-klinkstatus_src_plugins_scripting_scripts_CMakeLists.txt,v 1.1 2016/04/15 14:25:49 jperkin Exp $

Explicitly disable optional ruby scripts, to avoid PLIST failures
if ruby happens to be installed.

--- klinkstatus/src/plugins/scripting/scripts/CMakeLists.txt.orig	2008-12-21 17:55:20.000000000 +0000
+++ klinkstatus/src/plugins/scripting/scripts/CMakeLists.txt
@@ -5,7 +5,7 @@ install(FILES
 macro_optional_find_package(RUBY)
 macro_log_feature(RUBY_FOUND "Ruby" "An Interpreted object-oriented scripting language" "http://www.ruby-lang.org" FALSE "" "For KLinkStatus example ruby scripts")
 
-if(RUBY_FOUND)
+if(notRUBY_FOUND)
     
     include_directories(${RUBY_INCLUDE_PATH})
 
